omega/omega-transport/omega-transport-resttemplate/src/main/java/org/apache/servicecomb/pack/omega/transport/resttemplate/RestTemplateConfig.java

```diff
@@ -18,25 +18,22 @@
 
 package org.apache.servicecomb.pack.omega.transport.resttemplate;
 
-import java.util.List;
-
+import org.apache.servicecomb.pack.omega.context.OmegaContext;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.web.client.RestTemplateBuilder;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.http.client.ClientHttpRequestInterceptor;
 import org.springframework.web.client.RestTemplate;
 
-import org.apache.servicecomb.pack.omega.context.OmegaContext;
-
 @Configuration
 public class RestTemplateConfig {
 
   @Bean(name = "omegaRestTemplate")
-  public RestTemplate omegaRestTemplate(@Autowired(required=false) OmegaContext context) {
```

Comment:
We should keep this method and add @ConditionalOnMissingBean

Reply:
You mean `@ConditionalOnMissingBean(RestTemplateBuilder.class)` ?

Reply:
RestTemplateBuilder is in the default dependency of spring boot, I think it is OK we keep the code here.